### ISSUE TITLE
euphoria-core: sort operator

### ISSUE THREAD

**Issue by xitep** [seznam/euphoria#40]
We'd like to:

```java
Sort.named("SORT")
    .of(input)
    .by(e -> Comparable)
    .setPartitioning(new RangePartitioning())
    .output();
```

Range partitioning by the given `.by` function and sorting by the same attribute within each partition.

**Comment by horkyada** [comment]
PR https://github.com/seznam/euphoria/pull/99

**Comment by vanekjar** [comment]
Resolved in PR #99
